code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1727,6 +1727,19 @@
 		src.desc = "A wish come true!"
 		reagents.add_reagent(NUTRIMENT, 8)
 
+/obj/item/weapon/reagent_containers/food/snacks/avocadosoup
+	name = "Vegetable soup"
+	desc = "May be served either hot or cold."
+	icon_state = "avocadosoup"
+	trash = /obj/item/trash/snack_bowl
+	food_flags = FOOD_LIQUID
+
+/obj/item/weapon/reagent_containers/food/snacks/avocadosoup/New()
+	..()
+	reagents.add_reagent(NUTRIMENT, 8)
```

Comment:
A typical avocado serving is moderate to rich in several B vitamins and vitamin K, with good content of vitamin C, vitamin E, and potassium.
They can also be an important source of healthy fat in diets with limited access to other fatty foods such as high-fat meats and fish, or dairy products.
Additionally, avocado consumption has been shown to lower blood cholesterol levels.